src/app.js

```diff
@@ -20,13 +22,26 @@ const clientId = process.env.GITHUB_CLIENT_ID || config.GITHUB_CLIENT_ID;
 const clientSecret = process.env.GITHUB_CLIENT_SECRET || config.GITHUB_CLIENT_SECRET;
 const sessionSecret = process.env.SESSION_SECRET || config.SESSION_SECRET;
 const callbackUrl = process.env.CALLBACK_URL || config.CALLBACK_URL;
+const sentryUrl = process.env.SENTRY_DSN;
 const ss = require('socket.io-stream');
-
-var errorHandler;
-var app = express();
-var server = require('http').Server(app);
-var io = require('socket.io')(server);
-var id = 0;
+const Raven = require('raven');
+const redisClient =  require('redis').createClient(process.env.REDIS_URL);
+const Queue = require('bee-queue');
+const queue = new Queue('generator-queue', {redis: redisClient});
+const app = express();
+// eslint-disable-next-line new-cap
+const server = require('http').Server(app);
+const io = require('socket.io')(server);
+let parsedCookie, sid, folder;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'folder' should be initialized on declaration.](https://app.codacy.com/manual/triplez/open-event-webapp/pullRequest?prid=3226522)